main.nf

```diff
@@ -698,7 +698,7 @@ process sample_correlation {
 
     script: // This script is bundled with the pipeline, in NGI-RNAseq/bin/
     def rlocation = params.rlocation ?: ''
-    prefix = input_files[0].toString() - ~/(_R1)?(_trimmed)?(_val_1)?(\.fq)?(\.fastq)?(\.gz)?$/
+    prefix = input_files[0].toString() - 'Aligned.sortedByCoord.out_gene.featureCounts.txt' 
```

Comment:
The [Travis test](https://travis-ci.org/SciLifeLab/NGI-RNAseq/builds/210908037) shows the following tag:
```
sample_correlation (SRR4238355_subsamp.sorted_gene.featureCounts.txt)
```
Is the above search / remove string correct?